frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/FieldListDrawer/field-panels/PaymentsInputPanel/ProductItem.tsx

```diff
@@ -0,0 +1,210 @@
+import { BiDotsHorizontalRounded, BiEditAlt, BiTrash } from 'react-icons/bi'
+import {
+  Box,
+  ButtonGroup,
+  Divider,
+  Drawer,
+  DrawerBody,
+  DrawerContent,
+  DrawerOverlay,
+  Flex,
+  Table,
+  TableContainer,
+  Tbody,
+  Td,
+  Text,
+  Tr,
+  useDisclosure,
+} from '@chakra-ui/react'
+
+import { Product } from '~shared/types'
+import { centsToDollars } from '~shared/utils/payments'
+
+import { useIsMobile } from '~hooks/useIsMobile'
+import Button from '~components/Button'
+import IconButton from '~components/IconButton'
+
+export const ProductItem = ({
+  product,
+  onEditClick,
+  onDeleteClick,
+  isDisabled,
+}: {
+  product: Product
+  onEditClick: () => void
+  onDeleteClick: () => void
+  isDisabled: boolean
+}) => {
+  const isMobile = useIsMobile()
+  return (
+    <>
+      <Box px="1rem" py="1rem" backgroundColor={'#F8F9FD'}>
+        <Flex justifyContent="center" alignItems="center">
+          <Box flexGrow={1}>
+            <Flex justifyContent="space-between">
+              <Text textStyle="subhead-1" pb="0.25rem" color="secondary.500">
+                {product.name}
+              </Text>
+              {isMobile && (
+                <MobileProductItemMenu
+                  isDisabled={isDisabled}
+                  onDeleteClick={onDeleteClick}
+                  onEditClick={onEditClick}
+                />
+              )}
+            </Flex>
+            <TableContainer>
+              <Table
+                style={{
+                  borderCollapse: 'separate',
+                  borderSpacing: '0 0',
+                }}
+              >
+                <Tbody>
+                  <Tr>
+                    <Td
```

Comment:
/supernit Might want to consider if this section could be extracted into something like this 
```jsx
<TRowContent 
  title="Amount"
  content={centsToDollars(product.amount_cents)}
/>
{product.multi_qty && (
  <TRowContent 
    title="Quantity limit"
    content={`between ${product.min_qty} to ${product.max_qty}`}
  />
)}
```

Reply:
Nice! Thanks for the suggestion Ken! [dbb16ad](https://github.com/opengovsg/FormSG/pull/6699/commits/dbb16ad8e3843769eae761a8db29be2c557d54fa)